packages/jaeger-ui/src/components/TracePage/TraceTimelineViewer/SpanDetail/KeyValuesTable.tsx

```diff
@@ -95,6 +93,8 @@ function formatValue(key: string, value: any) {
         }}
       />
     );
+  } else {
+    content = stringMarkup(parsed);
   }
```

Comment:
I think we should continue checking type for number and Boolean and use the corresponding styles.  I would rename `stringMarkup` to `scalarMarkup` and do that check internally in order to switch between `json-markup-string`, `json-markup-number`, and `json-markup-bool`. This way the styling will be consistent with  the JSON rendering, but without extra indentation.